zzk/service/poollistener.go

```diff
@@ -127,6 +127,26 @@ func (l *PoolListener) Spawn(shutdown <-chan interface{}, poolID string) {
 				logger.WithError(err).Warn("Error Syncing")
 				timeout.Reset(l.Timeout)
 			}
+
+			select {
+			case <- node.ResourcePool.ReassignVIPS:
```

Comment:
I don't understand what this is doing.  At what point is the master reassigning the ip?